### PR TITLE
Bugfixes and stuff

### PR DESCRIPTION
**Pull request by tckmn**
- make it work with the latest version of `cfractal`
- fix a thing with `unique_ips.log` (I was going to remove it, but it's easy to fix)
- update the README

### CODE REVIEW

File: server [hunk 1]
```diff
@@ -61,7 +61,8 @@ Not enough variations
 END
             return 0;
         fi
-        ./fractal_gen -w $width -h $height $fracargs > /tmp/fractal$$.ppm;
+        ./fractal_gen -w $width -h $height $fracargs
+        mv out000.ppm /tmp/fractal$$.ppm
```
Comment (0e4ef622) [inline server:65]: What if two people are requesting a fractal simultaneously? There would be two instances of cfractal trying to write to the same file.

File: README.md [hunk 2]
```diff
@@ -3,5 +3,8 @@ cfractal-server
 
 Before you run the server, compile cfractal
 
+    $ git clone https://github.com/KeyboardFire/cfractal
```
Comment (0e4ef622) [inline README.md:6]: Wouldn't it be better to use `git submodule` to make sure that the version of cfractal used works with the server?

File: server (hunk 1 above)
Comment (tckmn) [inline server:65]: cfractal doesn't support specifying the output file yet. Working on it...

File: README.md (hunk 2 above)
Comment (tckmn) [inline README.md:6]: I'll try not to make breaking changes from here on out, but possibly. I've never used `git submodule` before ¯_(ツ)_/¯

**Pull request merged**